### PR TITLE
Skillset resource api

### PR DESCRIPTION
**Pull request by SundayAdefila**

### CODE REVIEW

**Comment by SundayAdefila** [comment]
@ceemion another one for you man!

**Comment by ceemion** [comment]
I thought we were not generating a model for Skillset again???

File: spec/models/skillset_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    end
+
+    it "should ensure the uniqueness of title" do
+      skill_1 = FactoryGirl.create(:skillset)
+      skill = FactoryGirl.build(:skillset, title: skill_1.title)
+      expect(skill).not_to be_valid
+      expect(skill.errors[:title]).to be_present
+    end
+  end
+end
```
Comment (ceemion) [inline spec/models/skillset_spec.rb]: You checked for when title and description are absent. In these cases there is an error. Other scenarios not here are:
1. When there is a valid title, and of what format?
2. When there is a valid description. Any restrictions on the length of text?

Comment (SundayAdefila) [inline spec/models/skillset_spec.rb]: resolved. @ceemion

Comment (ceemion) [inline spec/models/skillset_spec.rb]: Thanks @SundayAdefila

**Pull request merged**